pkg/util/file/file.go

```diff
@@ -42,3 +44,37 @@ func CopyFile(srcFile, dstFile string) (err error) {
 	}
 	return dFile.Sync()
 }
+
+// GenAbsFilePath return all the path with a given file name
+func GenAbsFilePath(baseDir, file string) (string, error) {
```

Comment:
Gen -> generate

Gen could also mean generic?

Function names can be longer if they improve readability.